### ISSUE TITLE
Nothing there?

### ISSUE THREAD

**Issue by darkpixel** [uncommitted-and-forgotten/django-uptimerobot#3]
The project is just a Django app template with no real functioning code.  Perhaps it should be deleted since it's the first result on Google for "django uptime robot" and the second result for "python uptime robot"--that way people don't go through the trouble of running through the install instructions only to find out it's actually an empty project.

**Comment by philippeowagner** [comment]
Good point. We expected to split the implementation for Django (TemplateTags, admin, management command, ..) from  the Python library and have https://github.com/arteria/python-uptimerobot as dependency. Not sure if this will happen because we stopped use Uptime Robot due to issues with the service. I will add a remark to the README.

**Comment by philippeowagner** [comment]
BTW: If you want to continue the development of django-uptimerobot or [python-uptimerobot](https://github.com/arteria/python-uptimerobot) feel free to contact me.

**Comment by darkpixel** [comment]
I created a fork after posting the issue and I'm working on solving it.  ;)

**Comment by philippeowagner** [comment]
@darkpixel any updates from your side?

**Comment by darkpixel** [comment]
Yeah, I'm slowly building it along with the project I intend to use it with, although there has been some recent discussion about dropping Uptime Robot.  Assuming I don't get told to drop it, it'll probably be 2-3 months before it's ready to be pushed back.
